.github/workflows/add-to-project.yaml

```
@@ -0,0 +1,26 @@
+name: Add issue to project
+
+on:
+  workflow_call:
+    inputs:
+      project_id:
+        type: number
+        required: true
+jobs:
+  add-issue-to-project:
+    name: Add issue with `status/triage` label to project
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'status/triage' }}
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@v1.2.1
+        with:
+          # Organization Github App ID and Private Key with write permissions to organization project
+          app_id: ${{ vars.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
```

We already have an organization-wide github app (we use this in multiple repo to sign and push commits), therefore we can reuse it here. 🙂 

Please change to:

```suggestion
          app_id: ${{ vars.JANUS_IDP_GITHUB_APP_ID }}
          private_key: ${{ secrets.JANUS_IDP_GITHUB_APP_PRIVATE_KEY }}
```

FFR: This app is organization-wide and those credentials are available also organization-wide (through organization secrets/variables).

---

Just want to confirm that it has write permissions to org projects right?

---

It has now 🙂